bedrock/firefox/templates/firefox/features/adblocker.html

```diff
@@ -0,0 +1,216 @@
+          <p>{{ _('Most ad blockers work in two ways.') }}</p>
+          <p>{{ _('First, they block communication from ad servers or others. Ads simply can’t get through.') }}</p>
+          <p>{{ _('Second, they hide parts of a web page that could be ads-- any section of a site that’s labeled "ad," for example. Most ad blockers use public and up-to-date lists of known ad servers.') }}</p>
+        </div>
+
+        <div class="mzp-c-card">
+          <h2 class="mzp-c-card-feature-title">{{ _('The Firefox ad blocker') }}</h2>
+          <p>{{ _('Firefox doesn’t include ad blocking out of the box, but it does include <a href="https://blog.mozilla.org/firefox/tracking-protection-always-on/">Tracking Protection</a>, which stops sites from installing those tracking cookies we talked about.') }}</p>
+          <p>It’s <a href="https://support.mozilla.org/en-US/kb/tracking-protection">easy to turn on</a>, and has the added benefit of making Firefox significantly faster.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-c-hero">
+    <div class="mzp-l-content l-content-narrow">
+      <div class="mzp-c-hero-body">
+        <h1 class="mzp-c-hero-title">{{ _('Why use ad blockers?') }}</h1>
+        <div class="mzp-c-hero-desc">
+          <p>{{ _('Ad blockers can also be used to make the web faster.') }}</p>
+          <p>{{ _('Ads can slow down sites while they load, or get in the way of a streamlined browsing experience.') }}</p>
+          <p>{{ _('Ads can also be flashy and annoying, distracting you from the stuff you really want to see.') }}</p>
+          <blockquote>{{ _('"Ads can slow down sites... or get in the way of a streamlined browsing experience."') }}</blockquote>
+          <p>
+          {% trans %}
+          Finally, ad companies can track your movements online. It’s called "re-targeting" and it works like this: You visit a site and click on a product. That site puts a small piece of code in your browser called a cookie. When you surf the web, sites will know you clicked on that product and they’ll display ads for it.
+          {% endtrans %}
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-c-hero">
+    <div class="mzp-l-content l-content-narrow">
+      <div class="mzp-c-hero-body">
+        <h1 class="mzp-c-hero-title">{{ _('Why you might not use an ad blocker') }}</h1>
+        <div class="mzp-c-hero-desc">
+          <p>
+            {% trans %}
+            Many media and news organizations make most of their money from advertising. If you block ads on their sites, they won’t get paid. If enough people block ads, business could suffer, and so will the quality of reporting and writing. Luckily, most ad blockers let you make exceptions for your favorite sites.
+            {% endtrans %}
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-c-card-feature mzp-has-aspect-3-2 mzp-l-card-feature">
```

Comment:
This is just a single image, so i think it's fine to leave it as that. It doesn't need to be a feature card component.